less/navigation.less

```diff
@@ -74,7 +73,8 @@
         .dropdown-menu-right();
 
         .dropdown-menu {
-          display: none;
+          margin-top: 0;
+          padding-top: 7px;
```

Comment:
I've removed the top margin for the nested dropdown element because this created a 2px zone where hovering over would cause the dropdown to close (which was annoying). It should visually look the same because I've merged this top margin into the top padding.